pkg/plugins/github/pullrequest.go

```diff
@@ -312,9 +345,99 @@ func (g *Github) getRemotePullRequest() error {
 	}
 
 	if len(query.Repository.PullRequests.Nodes) > 0 {
-		g.remotePullRequest = query.Repository.PullRequests.Nodes[0]
+		p.remotePullRequest = query.Repository.PullRequests.Nodes[0]
 		return nil
 	}
 
 	return nil
 }
+
+// getPullRequestLabelsInformation queries GitHub Api to retrieve every labels assigned to a pullRequest
+func (p *PullRequest) GetPullRequestLabelsInformation() ([]repositoryLabelApi, error) {
+
+	/*
+		query getPullRequests(
```

Comment:
I know this is only a comment, but the query indentation is kinda broken here

Reply:
How would you improve the situation ?

Reply:
I would reindent it? I don't know if it's a problem really worth it tbh

Reply:
Sorry for misreading the comment, I just fix the indentation